src/blocks/RulesBlock.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+function RulesModuleHeader() {
```

Comment:
Just a style note: the codebase uses `const RulesModuleHeader = () => {}` as as style. Mostly a personal preference.